.env

```diff
@@ -1,15 +1,53 @@
-# Frontend ENV Variables
+# For more detailed description of the ENV variables please visit: https://docs.chaosgenius.io/docs/Operator_Guides/Configuration/Config%20Parameters
+
+# User Configurable Variables
+## Webapp URL
+CHAOSGENIUS_WEBAPP_URL=http://localhost:8080/  # URL of the Chaos Genius deployment. Usually, this will be http://<ip-address-or-hostname>:8080/ (http://localhost:8080/ in local installations).
```

Comment:
I think it would look better and be easier to edit if the description was above each line and not on the same line